Collaborator-Guide.md

```diff
@@ -18,21 +18,90 @@ current release stream.
 Use the correspondingly named branch, e.g. `5.0`, for anything intended for
 a future release of Express.
 
-## Steps for contributing
+## Submitting a Pull Request (PR)
 
-1. [Create an issue](https://github.com/expressjs/express/issues/new) for the
+Before you submit your Pull Request (PR) consider the following guidelines:
+
+1. Search [GitHub](https://github.com/expressjs/express/pulls) for an open or closed PR that relates to your submission. You don't want to duplicate effort.
+
+2. [Create an issue](https://github.com/expressjs/express/issues/new) for the
    bug you want to fix or the feature that you want to add.
-2. Create your own [fork](https://github.com/expressjs/express) on GitHub, then
-   checkout your fork.
-3. Write your code in your local copy. It's good practice to create a branch for
-   each new issue you work on, although not compulsory.
-4. To run the test suite, first install the dependencies by running `npm install`,
-   then run `npm test`.
-5. Ensure your code is linted by running `npm run lint` -- fix any issue you
-   see listed.
-6. If the tests pass, you can commit your changes to your fork and then create
-   a pull request from there. Make sure to reference your issue from the pull
-   request comments by including the issue number e.g. `#123`.
+
+3. Fork the `expressjs/express` repo.
+
+4. Make your changes in a new git branch:
+
+ ```shell
+   git checkout -b my-fix-branch master
```

Comment:
Can you clarify at what indent level you are intending these blocks to be at? They start at two spaces, then a couple down just one space, and then ended with no spaces.